startdoc.sh

```
@@ -0,0 +1,28 @@
+DOCKERFILE_NAME="Dockerfile"
+
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" == "" ]]; then
+  echo "🔧 Docker image not found. Building image..."
+  docker build -t $IMAGE_NAME -f $DOCKERFILE_NAME .
+else
+  echo "✅ Docker image already exists."
+fi
+
+if [[ "$(docker ps -aq -f name=$CONTAINER_NAME)" == "" ]]; then
+  echo "🚀 Starting new container..."
+  docker run -it --name $CONTAINER_NAME -v $(pwd):/wave $IMAGE_NAME
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)